users/README.md

```diff
@@ -129,6 +130,30 @@ Creates a new User.
   - **Code:** 401
     - **Content:** `{ 'statusCode': 401, 'error': 'Unauthorized', 'message': 'Unauthenticated User' }`
 
+## **PATCH /users/self**
+
+Updates data of the User.
+
+- **Params**  
+  None
+- **Query**  
+  None
+- **Headers**  
+  Content-Type: application/json
+- **Cookie**  
+  rds-session: `<JWT>`
+- **Body** `{ <user_object> }`
+- **Success Response:**
+  - **Code:** 200
```

Comment:
We should be sending in a 204 for this PATCH request

Reply:
Reference: [MDN](https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/204#:~:text=The%20HTTP%20204%20No%20Content,included%20in%20such%20a%20response.)

Reply:
Changes as per the reveiw done. Thank you!